lib/config.go

```diff
@@ -327,7 +327,7 @@ func (c *Config) UpdateFromFile(path string) error {
 
 	_, err = toml.Decode(string(data), t)
 	if err != nil {
-		return err
+		return fmt.Errorf("unable to decode configuration %v: %v", path, err)
```

Comment:
could we use `errors.Wrapf`?

Reply:
Oh yeah sure we could! Unfortunately it has been merged but I will change it if I come there again. :)